docs/ruleset_functions/get_zone_conditioning_category.md

```diff
@@ -19,21 +19,23 @@ Constants:
+- For each HVAC system id in dictionary: `for hvac_sys_id in hvac_zone_list_w_area_dict.keys():` (Note XC, this only gets HVAC systems serving zones. Orphan HVAC systems are not looped)
 
-- Get dictionary for the list of zones and their total floor area served by each HVAC system in RMR: `hvac_zone_list_w_area_dict = get_hvac_zone_list_w_area(RMR)`
+  - Get total central sensible cooling capacity for HVAC system:
+  **[CH: The hvac system has at most one cooling_system, one heating_system, and one preheat_system. Also, I think we need to handle any of these systems missing.]**
+   `total_central_sensible_cool_capacity = SUM(cooling_system.sensible_cool_capacity for cooling_system in GET_COMPONENT_BY_ID(hvac_sys_id).cooling_system)`
 
-  - For each HVAC system id in dictionary: `for hvac_sys_id in hvac_zone_list_w_area_dict.keys():` (Note XC, this only gets HVAC systems serving zones. Orphan HVAC systems are not looped)
+  - Get total central heating capacity for HVAC system: `total_central_heat_capacity = SUM(heating_system.heat_capacity for heating_system in GET_COMPONENT_BY_ID(hvac_sys_id).heating_system) + SUM(preheat_system.heat_capacity for preheat_system in GET_COMPONENT_BY_ID(hvac_sys_id).preheat_system)`
 
-    - Get total central sensible cooling capacity for HVAC system: `total_central_sensible_cool_capacity = SUM(cooling_system.sensible_cool_capacity for cooling_system in GET_COMPONENT_BY_ID(hvac_sys_id).cooling_system)`
+  - Calculate and save total central sensible cooling output per floor area for HVAC system to dictionary: `hvac_cool_capacity_dict[hvac_sys_id] = total_central_sensible_cool_capacity / hvac_zone_list_w_area_dict[hvac_sys_id]["TOTAL_AREA"]`
 
-    - Get total central heating capacity for HVAC system: `total_central_heat_capacity = SUM(heating_system.heat_capacity for heating_system in GET_COMPONENT_BY_ID(hvac_sys_id).heating_system) + SUM(preheat_system.heat_capacity for preheat_system in GET_COMPONENT_BY_ID(hvac_sys_id).preheat_system)`
+  - Calculate and save total central heating output per floor area for HVAC system to dictionary: `hvac_heat_capacity_dict[hvac_sys_id] = total_central_heat_capacity / hvac_zone_list_w_area_dict[hvac_sys_id]["TOTAL_AREA"]`
 
-    - Calculate and save total central sensible cooling output per floor area for HVAC system to dictionary: `hvac_cool_capacity_dict[hvac_sys_id] = total_central_sensible_cool_capacity / hvac_zone_list_w_area_dict[hvac_sys_id]["TOTAL_AREA"]`
+- Get building climate zone: `climate_zone = RMR.weather.climate_zone`  
 
-    - Calculate and save total central heating output per floor area for HVAC system to dictionary: `hvac_heat_capacity_dict[hvac_sys_id] = total_central_heat_capacity / hvac_zone_list_w_area_dict[hvac_sys_id]["TOTAL_AREA"]`
+- Get heated space criteria: `system_min_heating_output = data_lookup(table_3_2,climate_zone)`
 
 - For each zone in RMR: `for zone in RMR...zones:`
 
```

Comment:
I will need to include zonal cooling output in the calculation below. The schema has been updated to reflect that.

Reply:
This is updated.

Reply:
This is updated.